install.sh

```diff
@@ -208,9 +175,28 @@ else
 			ENABLE_USB_CAM=false
 		fi
 	fi
+
+	# Give users the option to install the CUDA toolkit if an Nvidia card is detected
```

Comment:
Within a degree of reason. It works better than all of the crazy things that needed to be done in order to use the driver on Ubuntu 14.04 and @DSsoto has had no issues with it for the past month or so. I would still just install the version managed by the Ubuntu package maintainers, myself. That's why I made this optional.